x-pack/plugins/graph/server/sample_data/ecommerce.ts

```diff
@@ -0,0 +1,391 @@
+    {
+      weight: 0.0040928391377725235,
+      width: 5.752419052533403,
+      inferred: false,
+      source: 10,
+      target: 5,
+    },
+    {
+      weight: 0.003998804111234147,
+      width: 2.741475956722648,
+      inferred: false,
+      source: 5,
+      target: 6,
+    },
+    { weight: 0.0000201191575509262, width: 2, inferred: false, source: 2, target: 1 },
+    { weight: 0.0019559590149107486, width: 2, inferred: false, source: 4, target: 7 },
+    {
+      weight: 0.005399134008600699,
+      width: 7.588395315032752,
+      inferred: false,
+      source: 10,
+      target: 10,
+    },
+    { weight: 0.0008406249972756651, width: 2, inferred: false, source: 11, target: 0 },
+    { weight: 0.002434040312854235, width: 2, inferred: false, source: 2, target: 7 },
+    { weight: 0.0007632277713300751, width: 2, inferred: false, source: 4, target: 0 },
+    { weight: 0.007114987799732724, width: 10, inferred: false, source: 9, target: 7 },
+    { weight: 0.00029149607092423423, width: 2, inferred: false, source: 4, target: 5 },
+    {
+      weight: 0.004628005825697707,
+      width: 3.2924126381437944,
+      inferred: false,
+      source: 0,
+      target: 9,
+    },
+    { weight: 0.0001769629690348846, width: 2, inferred: false, source: 0, target: 2 },
+    {
+      weight: 0.0017862657198589743,
+      width: 2.510567509231816,
+      inferred: false,
+      source: 11,
+      target: 11,
+    },
+    {
+      weight: 0.0023385207220538266,
+      width: 3.286752961321555,
+      inferred: false,
+      source: 3,
+      target: 3,
+    },
+    { weight: 0.0005977285667016662, width: 2, inferred: false, source: 6, target: 1 },
+    {
+      weight: 0.00523765988442745,
+      width: 7.361446051424297,
+      inferred: false,
+      source: 6,
+      target: 6,
+    },
+  ],
+  urlTemplates: [
+    {
+      url:
+        '/app/kibana#/discover?_a=(columns%3A!(_source)%2Cindex%3Aff959d40-b880-11e8-a6d9-e546fe2bba5f%2Cinterval%3Aauto%2Cquery%3A(language%3Akuery%2Cquery%3A{{gquery}})%2Csort%3A!(_score%2Cdesc))',
+      description: 'Raw documents',
+      isDefault: true,
+      encoderID: 'kql-loose',
+    },
+  ],
+  exploreControls: {
+    useSignificance: false,
+    sampleSize: 2000,
+    timeoutMillis: 5000,
+    maxValuesPerDoc: 1,
+    minDocCount: 3,
+  },
+  indexPatternRefName: 'indexPattern_0',
+};
+
+export function registerEcommerceSampleData(sampleDataRegistry: SampleDataRegistrySetup) {
+  sampleDataRegistry.addSavedObjectsToSampleDataset(datasetId, [
+    {
+      type: 'graph-workspace',
+      id: '46fa9d30-319c-11ea-bbe4-818d9c786051',
+      version: '2',
+      attributes: {
+        title: 'Kibana Sample Data - eCommerce',
+        description:
+          'This is a sample graph based on an eCommerce data set.  It shows the gender, continent, and product category of purchases.  The thicker the line is, there are more correlated documents between the vertices.',
```

Comment:
I'm not 100% sure, but "The thicker the line is, there are more correlated documents between the vertices." sounds a bit like german (also like austriakisch)

Reply:
Hehe, ping @shaunmcgough - these are your German roots shining through.